panimg/image_builders/dicom.py

```diff
@@ -344,6 +344,27 @@ def read(self) -> SimpleITKImage:
         )
 
 
+def _sort_studies(studies, file_errors):
```

Comment:
This is filtering, not sorting, right? So `_filter_studies`, and `unsortable_studies` could be `ignored_studies` etc? The function could do with a docstring to let others know its intent.

Reply:
The function is sorting the items in each individual study while filtering out studies of which the items cannot be sorted. I've added a docstring and renamed the function in an attempt to make this clearer.

Reply:
Ah I see now, much clearer, thanks!